Example/LayoutInspectorExample/AppDelegate.swift

```diff
@@ -17,6 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
         LayoutInspector.shared.setAutoTrigger(.screenshot)
+        LayoutInspector.shared.setAutoTrigger(.shake)
```

Comment:
@miku1958 this line overwrites the line above, and thus layout won't be shown on taking screenshot. 
I think it worth adding either control to allow selection of preferred trigger type or implement OptionSet which will allow having multiple enabled trigger types at one time